parquet/parquet.cpp

```diff
@@ -1497,6 +1497,12 @@ static void dissect_field(pq_writer *pw, lua_State *lua, pq_node *n, int16_t r,
       }
     }
     break;
+  case LUA_TLIGHTUSERDATA:
+    if (lua_touserdata(lua, -1) != NULL) {
+      add_value(lua, n, c, r, n->dl);
+      break;
+    }
+    /* fall through */
```

Comment:
Use the special syntax `/* FALLTHRU */` see https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html